app/scripts/utils/functional.es6

```diff
@@ -82,7 +83,7 @@ const countByProp = (propName, agg, item) => {
 };
 
 const sumByProp = (propName, agg, item) => {
-  if (isNumber(item[propName])) {
+  if (isNumber(item[propName]) && !isNaN(item[propName])) {
     if (isUndefined(agg.total)) {
       agg.total = 0;
```

Comment:
no, we shall ignore it.
The problem is that there where a lot of NaN in the parsing and since NaN + anything = NaN there was anything to draw.